src/chainstate/stacks/boot/pox-testnet.clar

```diff
@@ -7,7 +7,7 @@
 (define-constant PREPARE_CYCLE_LENGTH u30)
 
 ;; Default length of the PoX reward cycle, in burnchain blocks.
-(define-constant REWARD_CYCLE_LENGTH u120)
+(define-constant REWARD_CYCLE_LENGTH u150)
```

Comment:
Should we have the same name for our constants in Clarity and Rust? (`REWARD_CYCLE_LENGTH ` vs `POX_ REWARD_CYCLE_LENGTH`)?